components/MetamaskChecker/index.js

```diff
@@ -57,7 +57,10 @@ class MetamaskChecker extends Component {
     try{
       const accounts = await window.web3js.eth.getAccounts();
       if(accounts && accounts.length > 0){
-        let balance = await window.web3js.eth.getBalance(accounts[0]);
+        let balance;
+        while(!balance){
```

Comment:
Damn, that's a way of brute forcing our way into a variable. Why we need a `while` if we have already `await` ?